CONTRIBUTING.md

```diff
@@ -1,172 +1,76 @@
+   2. If the issue you want addressed is a specific proposal or a bug, then open a [new issue](https://github.com/cosmos/gaia/issues/new/choose).
+   3. Review existing [issues](https://github.com/cosmos/gaia/issues) to find an issue you'd like to help with.
+3. Participate in thoughtful discussion on that issue.
+4. If you would like to contribute:
+   1. Ensure that the proposal has been accepted.
+   2. Ensure that nobody else has already begun working on this issue. If they have,
+      make sure to contact them to collaborate.
+   3. If nobody has been assigned for the issue and you would like to work on it,
+      make a comment on the issue to inform the community of your intentions
+      to begin work.
+5. To submit your work as a contribution to the repository follow standard GitHub best practices. See [pull request guideline](#pull-requests) below.
+
+**Note:** For very small or blatantly obvious problems such as typos, you are
 not required to an open issue to submit a PR, but be aware that for more complex
 problems/features, if a PR is opened before an adequate design discussion has
-taken place in a github issue, that PR runs a high likelihood of being rejected.
-
-Take a peek at our [coding repo](https://github.com/tendermint/coding) for
-overall information on repository workflow and standards. Note, we use `make
-tools` for installing the linting tools.
-
-Other notes:
-
-- Looking for a good place to start contributing? How about checking out some
-  [good first issues](https://github.com/cosmos/gaia/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-- Please make sure to run `make format` before every commit - the easiest way
-  to do this is have your editor run it for you upon saving a file. Additionally
-  please ensure that your code is lint compliant by running `golangci-lint run`.
-  A convenience git `pre-commit` hook that runs the formatters automatically
-  before each commit is available in the `contrib/githooks/` directory.
-
-## Submissions
+taken place in a GitHub issue, that PR runs a high likelihood of being rejected 
+without review.
 
-Generally, when structuring a submission for the Cosmos ecosystem, the submission might span multiple layers
-of the technology stack, including the Cosmos SDK and Tendermint dependencies.
+## Teams Dev Calls
 
-### Structuring a submission
+Gaia has many stakeholders contributing and shaping the project. The Core Gaia team is composed of Informal Systems developers. Any long-term contributors and additional maintainers from other projects are welcome. We use self-organizing principles to coordinate and collaborate across organizations in structured "EPIC" that focus on specific problem domains or architectural components of Gaia.
```

Comment:
Is there a link that talks about what Epics are? I feel the term is uncommon enough that an explanation would help.

Maybe this? https://www.atlassian.com/agile/project-management/epics

Reply:
@mpoke Many contributors in the ecosystem do not come from traditional professional software development backgrounds and might not be familiar with concepts like agile/scrum/extreme programming workflows. Might be worth a section on the basic process workflow the team is using at present.

Reply:
I added a link to the project board, i.e., https://github.com/orgs/cosmos/projects/28/views/11